docs/source/tutorial/running_ablation.rst

```diff
@@ -1,3 +1,87 @@
 Running an Ablation Study
--------------------------
-.. automodule:: pykeen.ablation
+=========================
+In PyKEEN, we can define an ablation study in a configuration file ('file_name.json') or within our own program.
```

Comment:
throughout the tutorial, use code highlighting more effectively for things like file names (double backticks)